Global/Eclipse.gitignore

```diff
@@ -49,3 +49,6 @@ local.properties
 
 # Code Recommenders
 .recommenders/
+
+# Scala IDE specific (Scala & Java development for Eclipse)
```

Comment:
@shiftkey, please take a look. I've reverted a change to Scala.gitignore, so there is only a change to Eclipse.gitignore in this PR.

I'll rework Scala.gitignore in another PR: will it be OK to have all moves in a single PR, or should I remove lines from Scala.gitignore in one PR and add them to specific Global sections in other ones?